src/app/static/src/app/components/projects/Projects.vue

```diff
@@ -10,7 +10,12 @@
                 </span>
             </div>
             <div class="my-3 col-6 clearfix">
-                <input type="text" class="form-control" v-translate:placeholder="'projectName'" v-model="newProjectName">
+                <input type="text" class="form-control"
+                       v-translate:placeholder="'projectName'"
+                       v-model="newProjectName">
+                <div class="invalid-feedback d-block"
```

Comment:
If this is d-inline you can avoid the rest of the screen jumping around when the message shows, though I guess there's a danger of them colliding on smaller screens..